js/js-object-hw

```diff
@@ -0,0 +1,31 @@
+const menu = {
+  pepperoni: {
+    toppings: ['garlic', 'tomatoes', 'pepperoni'],
+    price: 4
+  }, 
+  cheese: {
+    toppings: ['Black Olives', 'Broccoli', 'Spinach'],
+    price: 6
+  },
+   hawaiian: {
+    toppings: ['Ham', 'Steak', 'Jalapenos'],
+    price: 8
+  },
+
+  margarita: {
+    toppings: ['Chicken', 'Ground Beef', 'Bacon Bits'],
+    price: 9
+  },
+  languica: {
+    toppings: ['Chick peas', 'Beef', 'Bacon'],
+    price: 10  
+  }
+}
+console.log(Object.keys(menu))
+
+if ('banana' in menu) {
+  console.log('There it is!')
+} else {
+  console.log('It aint there')
+}
+ 
```

Comment:
awesome! always appreciate a person using a classic tool like a conditional or a loop instead of a fancy function. 🥇 💯 🥳 

in case you're curious, there's also an object method called [`hasOwnProperty()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty)